neptunecontrib/monitoring/sklearn.py

```diff
@@ -1228,8 +1229,7 @@ def log_silhouette_chart(model, X, experiment=None, **kwargs):
 
 def _validate_experiment(experiment):
     if experiment is not None:
-        if not isinstance(experiment, neptune.experiments.Experiment):
-            ValueError('Passed experiment is not Neptune experiment. Create one by using "create_experiment()"')
```

Comment:
Wyrzucając to sprawdzenie rozluźniasz wcześniejsze wymagania co do obiektu,

Reply:
Przywróciłem starego check-a. Na początku robiłem podobnie jak tutaj dlatego widziałem sens wywalenia go.